dockerfiles/latest-trusty/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:14.04
+
+# all in one RUN to produce less, smaller layers
+RUN bash -c "\
+	sudo apt-get -qqy update; \
```

Comment:
Are you asking about the use of `;` vs `&&` ? You're right, it would be best to use `&&`. I'll update momentarily.